### ISSUE TITLE
error in Webinterface

### ISSUE THREAD

**Issue by psycodeliccircus** [Delivator/discord-bot#12]
in my already tried it all the more ways it returns with being I had not logged in.
could help me?

**Comment by Delivator** [comment]
Yeah sorry I should have mentioned it somewhere, but the webinterface isn't fully functional yet. Listing servers and the members on that server works. Getting a users access token via the login button works, but the bot isn't doing anything with that access token.
